modules/samples/product/FlickrClient/src/org/wso2/appserver/sample/flickr/client/FlickrServiceFrame.java

```diff
@@ -411,7 +411,7 @@
     private FlickrServiceClient client = new FlickrServiceClient();
     private String key;
     private String host = "api.flickr.com";
-    private String port = "80";
+    private String port = "443";
```

Comment:
if both host and port are the same for all FlickrServiceFrame objects, better making these as CONSTANTS rather than instance variables.